packages/generator/src/edmx-to-vdm/v4/entity.ts

```diff
@@ -65,26 +65,28 @@ export function generateEntitiesV4(
       classNames,
       formatter
     ),
-    functions: generateBoundOperations(
-      serviceMetadata,
-      entityMetadata.entityType.Namespace,
-      'function',
-      entities,
-      complexTypes,
-      formatter,
-      entityMetadata.entitySet.Name,
-      classNames[entityMetadata.entitySet.Name]
-    ),
-    actions: generateBoundOperations(
-      serviceMetadata,
-      entityMetadata.entityType.Namespace,
-      'action',
-      entities,
-      complexTypes,
-      formatter,
-      entityMetadata.entitySet.Name,
-      classNames[entityMetadata.entitySet.Name]
-    )
+    operations: [
+      ...generateBoundOperations(
+        serviceMetadata,
+        entityMetadata.entityType.Namespace,
+        'function',
+        entities,
+        complexTypes,
+        formatter,
+        entityMetadata.entitySet.Name,
+        classNames[entityMetadata.entitySet.Name]
+      ),
+      ...generateBoundOperations(
+        serviceMetadata,
+        entityMetadata.entityType.Namespace,
+        'action',
+        entities,
+        complexTypes,
+        formatter,
+        entityMetadata.entitySet.Name,
+        classNames[entityMetadata.entitySet.Name]
+      )
```

Comment:
[q] Is it still necessary to have to calls of `generateBoundOperations` here or can we merge them into one?